rialto_airflow/mais.py

```diff
@@ -31,7 +31,11 @@
 logger = logging.getLogger(__name__)
 
 
-def get_token(client_id: str, client_secret: str, base_url: str) -> Optional[str]:
+class TokenFetchError(LookupError):
+    pass
```

Comment:
waffled between adding this and just using `LookupError` directly.  happy to ditch the custom error if that's more pythonic or otherwise preferable.